customization.cfg

```diff
@@ -42,6 +42,7 @@ _force_all_threads="true"
 _noccache="false"
 
 # Set to true to use modprobed db to clean config from unneeded modules. Speeds up compilation considerably. Requires root - https://wiki.archlinux.org/index.php/Modprobed-db
+# Using this option can trigger user prompts of the config doesn't go smoothly.
```

Comment:
> prompts of the config

I think you meant `if` instead of `of` here.

Reply:
Absolutely x)

Reply:
I will changed to commit message with the double "t" too